ArduCopter/AP_Arming.cpp

```diff
@@ -603,7 +603,7 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
 
     // always check if the current mode allows arming
     if (!copter.flightmode->allows_arming(method)) {
-        check_failed(true, "Mode not armable");
+        check_failed(true, "%s Mode not armable", copter.flightmode->name());
```

Comment:
```suggestion
        check_failed(true, "Mode %s is not an armable mode", copter.flightmode->name());
```

A slight effort at changing it to read better, at a minimum we should try and avoid the double capitalization.

Reply:
@WickedShell's changed it to be more correct english but also makes it longer.

I don't really mind too much but I'd prefer this PR's original wording but with "Mode" changed to "mode" as MichaelDB suggest.

Reply:
> @WickedShell's changed it to be more correct english but also makes it longer.

As long as it fits inside of a single status text I don't care to much. I mostly want it to read well, because we have a tendency to hand users excessively terse messages there and I've watched people spend more time puzzling it out when we could make it a much better experience with a bit more verbiage. The key in this case I'm wondering about is you could read the sentence "RTL mode is not armable" and it's very easy as the user to go "yes,  the mode I'm in right now is refusing to arm for some reason, but I want to arm, why won't it?"